### ISSUE TITLE
Is 12 FPS a reasonable result for Nvidia K80

### ISSUE THREAD

**Issue by EvanMu96** [pjreddie/darknet#1401]
my configuration:
GPU=1
CUDNN=1
OPENCV=0
YOLOv3 416 image size 1280 x 720
yolov3-voc.cfg is my cfg, I just modified some filter number before yolo layers.
I worked on FloydHub(with Nvidia Tesla K80). 
And it takes about 0.07 average for test an image with the size 1280x720. I think it should have better efficiency than 12FPS around on Nvidia K80.
Is there anyone can help me? Many thanks.

**Comment by EvanMu96** [comment]
By the way. The training process has a normal GPU utilization.

**Comment by vaishnavm217** [comment]
YOLOv3 being big model and the given input size, it seems reasonable fps for me. You can increase the fps by:
- Reducing the image size (duh)
- Upgrading CUDA
- increasing threshold (i think it increases as the detections will be less)

**Comment by fzupang** [comment]
@EvanMu96 Hello! Have you solved the problem?I use K80 and my fps is about 12,too.

**Comment by EvanMu96** [comment]
> @EvanMu96 Hello! Have you solved the problem? I use K80 and my fps is about 12,too.

Yes. I test the same model with the same configuration on GTX1080 and it worked about 35FPS.
I guess that 10series GPU have a more modern GPU architecture so that it use less inference time.
Hope that will help you.

**Comment by rohitsaluja22** [comment]
Hey its cool, I also tried on GTX1080 with data/dog.jpg image, its of size 768x576.I got 17 fps (as boxes predicted in 0.056977 seconds), using cuda-8 and cudnn5.0

How you got 35 FPS? It would be helpful if you can reply back with following configs:-
image size:
cuda version:
cudnn version:
threshold: default or you modified...

Thanks in advance

**Comment by MichaelCong** [comment]
how to test fps?thanks

**Comment by EvanMu96** [comment]
> how to test fps?thanks

Basically you just calculate it by the number of images and the running time.

**Comment by othmane-kada** [comment]
@EvanMu96 hi, can you please tell me which Motherboard you used with K80?
